config/config.go

```diff
@@ -374,6 +397,31 @@ func NewConfig() (c *Config) {
 				WhiteListIstioSystem: []string{"jaeger-query", "istio-ingressgateway"},
 			},
 		},
+		HealthConfig: HealthConfig{
```

Comment:
Is this default config the previous existing logic for calculate health before this feature ?
Or are we introducing a new default here ?

Reply:
This is the default config that we had